squad/frontend/static/squad/controllers/metricThreshold.js

```diff
@@ -47,8 +57,16 @@ function MetricThresholdController($scope, Threshold) {
         var threshold_index = $scope.currentThreshold.index
```

Comment:
inconsistent leading spaces in this section

Reply:
Fixed this one too, there was actually a lot more lines like this one. Nice catch